NEWS.md

```diff
@@ -337,6 +337,9 @@ Library improvements
   * A new `Dates.Time` type was added that supports representing the time of day
     with up to nanosecond resolution ([#12274]).
 
+  * Raising one or negative one to a negative integer power no longer
+    throws a `DomainError` ([#18342]).
```

Comment:
This seems like it could use a little more elaboration – i.e. what does it now do instead of throwing an error?